src/EnergyPlus/SZVAVModel.cc

```diff
@@ -772,8 +773,11 @@ namespace SZVAVModel {
                         Par(9) = lowWaterMdot; // minCoilFluidFlow - low fan speed water flow rate > 0 if SAT limited
                         Par(12) = AirMassFlow; // sets air flow rate used when iterating on coil capacity
                         Par(13) = 0.0;         // other than 0 means to iterate on SA temperature
-
-                        TempSolveRoot::SolveRoot(state, 0.001, MaxIter, SolFlag, PartLoadRatio, FanCoilUnits::CalcFanCoilWaterFlowResidual, 0.0, 1.0, Par);
+                        if (SZVAVModel.HCoilType_Num == HCoil_Water || !HeatingLoad) {
```

Comment:
The arguments in this IF check are a difficult to fully grok.  Anytime there is not a heating load it goes into the original residual function?  Could you just clarify?

Reply:
(Though it's used many times in here so it's probably fine, but still clarifying the logic would help make it digestable)

Reply:
The residual function FanCoilUnits::CalcFanCoilWaterFlowResidual() various the water flow rate to meet the heating load for hot water heating coil. The other residual function CalcFanCoilLoadResidual() varies PLR to meet the heating load for electric heating coils. We cannot use the same residual function. The former residual function is used for both cooling and heating modes of operation. Chilled water cooled coil flow rate is varied to meet the cooling load using the same residual function and is captured with "!HeatingLoad".